test/integration/puppetlabs/services/master/tasks_int_test.clj

```diff
@@ -105,13 +106,22 @@
 
         (testing "the expected response body is returned"
           (let [expected-response {"metadata" metadata
+                                   "name"  task-name
                                    "files" [{"filename" "poc.sh"
                                              "sha256" "f24ce8f82408237beebf1fadd8a3da74ebd44512c02eee5ec24cf536871359f7"
                                              "size_bytes" 20
                                              "uri" {"path" "/puppet/v3/file_content/tasks/shell/poc.sh"
                                                     "params" {"environment" "production"}}}]}]
             (is (= expected-response (parse-response response)))))
 
+        (testing "the name of the init task is just the module's name"
+          (testutils/write-tasks-files "exec" "init" "eval \"$@\"" {"description" "it is"})
+          (is (= "exec"
+                 (-> (get-task-details "production" "exec::init")
+                     (doto println)
```

Comment:
Out of curiosity, what is the purpose of this line?

Reply:
Oops - left in from debugging